fiaas_deploy_daemon/config.py

```diff
@@ -228,6 +228,9 @@ def _parse_args(self, args):
         parser.add_argument("--use-apiextensionsv1-crd",
                             help="Use apiextensions/v1 CustomResourceDefinition instead of apiextensions/v1beta1",
                             action="store_true", default=False)
+        parser.add_argument("--deactivate-crd-creation",
```

Comment:
Nitpicking, but I think naming the flag `disable-crd-creation` would make it a bit more similar to some of the other config flags compared to `deactivate-crd-creation`, since there are a lot of `enable-$feature` flags